gtsfm/common/gtsfm_data.py

```diff
@@ -197,3 +210,17 @@ def from_selected_cameras(cls, gtsfm_data: "GtsfmData", camera_indices: List[int
                 new_data.add_track(track)
 
         return new_data
+
+    def get_scene_avg_reprojection_error(self) -> float:
+        """Get average reprojection error for all 3d points in the entire scene
+
+        Returns:
+            scene_avg_repoj_error: average of reprojection errors for every 3d point to its 2d measurements
+        """
+        scene_errors = []
```

Comment:
we're actually extending the list though, since each element returned is a numpy array, and it is cast to a list in the `extend()` call. and i dont know how to do that with list comprehension